lib/view.js

```diff
@@ -25,6 +25,17 @@ module.exports = Vue.extend({
             e.stopPropagation();
             e.preventDefault();
             return false;
+        },
+        createAppFromTemplate: function (templateId) {
+            var self = this;
+            self.$root.isReady = false;
+            var app = self.$root.storage.createApp({
+                template: templateId
+            });
+            setTimeout(function () {
```

Comment:
Can you use a setInterval to check whatever you're trying to ensure is ready? This looks like a potential race condition...